xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -455,11 +455,15 @@ The <xref:System.Text.Json.Utf8JsonWriter> will continue to use the original wri
 
 ## Remarks
 
+The maximum allowed size of the binary data to write as Base64 is 125_000_000 bytes (or ~125MB). Exceeding this limit will result in an <xref:System.ArgumentException> being thrown.
+
 The property name is escaped before writing.
```

Comment:
Wouldn't it look better if these sentences were combined into one @ahsonkhan? 

The property name is escaped and the bytes are encoded before writing.

Reply:
Sure, I am fine with either since I don't see it making too big a difference one way or another.

If we want to be consistent with some of the other APIs (like WriteString): https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonwriter.writestring?view=netcore-3.0#System_Text_Json_Utf8JsonWriter_WriteString_System_String_System_String_, we should combine them.

That has it as:
> The property name and value is escaped before writing.

So your suggestion looks good, @mairaw

Reply:
Up to you @lauxjpn, if you want to make the fix in this PR before we merge :)

Reply:
I'll change it. Is it really "is escaped" instead of "are escaped"?

Reply:
To my ear, both seem correct. We have other places we use "or" in which case "is" probably makes sense.

Like: `Thrown when the specified property name or value is too large.` rather than `...  are too large.`

In this case, I am not sure, and to me it doesn't seem worth it to fix. I'd leave it up to @mairaw /@rpetrusha to chime in on that, especially if using "is" in this case is incorrect.

Reply:
if or is used, the person of the verb depends on whether the second subject in the or clause is singular or plural (e.g. one or two is correct, one car or two cars are parked outside); if and, the verb is plural.

Reply:
Good to know. My english classes lay way in the post now. Though back in those days, "The property name and value escaped before writing." would have been the only accepted answer ;)

Reply:
> if and, the verb is plural.

So, if I understand your suggestion correctly, we should change the existing docs to use "are"

> The property name and value **are** escaped before writing.

Reply:
Correct. I can create a PR for that.